extensions/amp-viewer-assistance/amp-viewer-assistance.md

```diff
@@ -63,7 +63,7 @@ The `amp-viewer-assistance` extension currently has two functions that can be in
   </tr>
   <tr>
     <td class="col-fourty"><code>updateActionState</code></td>
-    <td>A function to send a message to the outer viewer representing a state change. Should contain an argument of the resulting state change.</td>
+    <td>A function to send a message to the outer viewer representing a state change. Should contain an <code>update</code> object as well as an <code>actionStatus</code> field denoting the resulting state change. Supported <code>actionStatus</code> strings are <code>ACTIVE_ACTION_STATUS</code>, <code>FAILED_ACTION_STATUS</code>, and <code>COMPLETED_ACTION_STATUS</code>.</td>
```

Comment:
An sample object might be clearer:

```
Must be invoked with a parameter named "update" whose value is a JSON object of the form:

{
  [required] "actionStatus": "ACTIVE_ACTION_STATUS" | "FAILED_ACTION_STATUS" | "COMPLETED_ACTION_STATUS"
  [optional] "result": <any>
}
```
Just made this up. There's probably a better way to specify JSON schema than this.

Reply:
Good idea, thanks for the suggestion.